core/arch/arm/plat-nuvoton/conf.mk

```diff
@@ -7,7 +7,6 @@ include core/arch/arm/cpu/cortex-armv8-0.mk
 CFG_ARM64_core = y
 endif #npcm845x
 CFG_USER_TA_TARGETS ?= ta_arm64
-$(call force,CFG_CORE_DYN_SHM,n)
```

Comment:
Please do not add this line in the first commit rather than removing it later...